analyze.py

```diff
@@ -382,6 +382,36 @@ def analyze_timeline(user_id, list_id, api, cache):
     users = fetch_users(timeline_ids, api, cache)
     return analyze_users(users, ids_fetched=len(timeline_ids))
 
+def analyze_my_timeline(user_id, api, cache):
+    # Timeline-functions are limited to 200 statuses
+    statuses = api.GetUserTimeline(screen_name=user_id, count=200, include_rts=True, trim_user=False, exclude_replies=False)
+    maxt = 0
```

Comment:
Why "maxt"? Perhaps "max_id", with a comment that since we're moving backwards in time, passing max_id skips newer tweets.

Reply:
Thanks! Pushed an edit with the variable name change and a comment.